Cargo.lock

```diff
@@ -2231,37 +2254,38 @@ dependencies = [
  "clap",
  "dirs 5.0.1",
  "fd-lock",
- "forc-tracing 0.67.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "fuel-asm",
+ "forc-tracing 0.67.0",
+ "fuel-asm 0.59.2",
  "hex",
  "paste",
  "regex",
  "serde",
  "serde_json",
  "serial_test",
  "sha2",
- "sway-core 0.67.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "sway-error 0.67.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "sway-types 0.67.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "sway-utils 0.67.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "sway-core 0.67.0",
```

Comment:
seems like theres still deps pointing to `0.67.0`?

Reply:
Good catch. I fixed this by updating `forc-utils` to 0.68 as well.